app/User.php

```diff
@@ -16,7 +16,7 @@ class User extends Authenticatable
      * @var array
      */
     protected $fillable = [
-        'name', 'email', 'password',
+        'name', 'email', 'password', 'church_id'
```

Comment:
It might make sense to turn users and churches into a many-to-many relationship so a user can rate multiple churches. We don't have to do this now though.